R/pkg/inst/tests/test_sparkSQL.R

```diff
@@ -108,6 +108,14 @@ test_that("create DataFrame from RDD", {
   expect_equal(count(df), 10)
   expect_equal(columns(df), c("a", "b"))
   expect_equal(dtypes(df), list(c("a", "int"), c("b", "string")))
+
+  localDF <- data.frame(name=c("John", "Smith", "Sarah"), age=c(19, 23, 18), height=c(164.10, 181.4, 173.7))
+  schema <- structType(structField("name", "string"), structField("age", "integer"), structField("height", "float"))
+  df <- createDataFrame(sqlContext, localDF, schema)
```

Comment:
OK. That is good. As #7311 is merged now. I should update this test case or it will fail due to this issue.